### PR TITLE
Fix opt.semgrep.import-text-template error

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: Sonia Singla <soniasingla.1812@gmail.com>

### CODE REVIEW

File: vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 	"path"
 	"strconv"
 	"strings"
-	"text/template"
+	"html/template"
```
Comment (haircommander) [inline vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go:14]: we shouldn't change this here. It should be changed in the source and revendored

Comment (soniasingla) [reply vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go:14]: where is the source @haircommander ? thanks

Comment (soniasingla) [reply vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go]: @haircommander so i changed in template and run `make vendor`

Comment (haircommander) [reply vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go]: you can find it based on where the vendor directory is:
github.com/containers/common/tree/master/pkg/apparmor/apparmor_linux.go

Comment (soniasingla) [reply vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go]: thanks @haircommander so I should make the PR in that repo and close this one? thanks

Comment (haircommander) [reply vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go]: yes I would recommend that, thanks :)

Comment (soniasingla) [reply vendor/github.com/containers/common/pkg/apparmor/apparmor_linux.go]: ok thanks @haircommander

**Pull request closed without merging**